### DISCUSSION TITLE
recursive search for dns/caa-fingerprint.yaml

### DISCUSSION THREAD

**Discussion by acbekoac** [projectdiscovery/nuclei-templates#7496] [Q&A]
the caa-fingerprint template finds "In CAA" records for "-u subdomain.domain.tld" but not if "-u www.subdomain.domain.tld" is used

**Comment by ehsandeep** [comment]
@acbekoac please include an example host, where CAA records exist for `www.subdomain.domain.tld` and nuclei failed to display match.
